src/escalus_fresh.erl

```diff
@@ -204,9 +205,16 @@ select(UserResources, FullSpecs) ->
     lists:filter(fun({Name, _}) -> lists:member(Name, UserNames) end,
                  FullSpecs).
 
+%% This is to ensure that there are no collisions between the same run
 fresh_int_suffix() ->
     integer_to_binary(erlang:unique_integer([monotonic, positive])).
 
+%% This is to ensure that there is no garbage data between runs
+timestamp_suffix() ->
+    {{_Y, M, D}, {H, Min, S}} = erlang:universaltime(),
+    iolist_to_binary(io_lib:format("~2..0w~2..0w~2..0w~2..0w~2..0w",
+                                   [M, D, H, Min, S])).
```

Comment:
Why not doing system_time and then calendar:system_time_to_rfc3999/2? Just thinking out loud 🤔